src/components/MeasurementOverlay/MeasurementOverlay.js

```diff
@@ -299,46 +345,63 @@ class MeasurementOverlay extends React.PureComponent {
   };
 
   render() {
-    const { annotation } = this.state;
+    const { annotation, position, transparentBackground } = this.state;
     const { isDisabled, t, isOpen } = this.props;
-    const className = getClassName('Overlay MeasurementOverlay', this.props);
     const key = mapAnnotationToKey(annotation);
 
     if (isDisabled || !annotation) {
       return null;
     }
 
-    if (this.shouldShowCustomOverlay(annotation)) {
-      // Get the props for this particular custom overlay
-      const customOverlayProps = this.props.customMeasurementOverlay.filter(customOverlay => customOverlay.validate(annotation))[0];
-      return (<CustomMeasurementOverlay annotation={annotation} {...customOverlayProps}/>);
-    }
-
-    if (key === 'ellipseMeasurement') {
-      return (<EllipseMeasurementOverlay annotation={annotation} isOpen={isOpen}/>);
-    }
-
     return (
-      <div className={className} data-element="measurementOverlay">
-        {this.renderTitle()}
-        <div className="measurement__scale">
-          {t('option.measurementOverlay.scale')}: {this.renderScaleRatio()}
-        </div>
-        <div className="measurement__precision">
-          {t('option.shared.precision')}: {annotation.Precision}
+      <Draggable
+        cancel="input"
+        position={position}
+        onDrag={this.syncDraggablePosition}
+        onStop={this.syncDraggablePosition}
+      >
+        <div
+          className={classNames({
+            Overlay: true,
+            MeasurementOverlay: true,
+            open: isOpen,
+            closed: !isOpen,
+            transparent: transparentBackground,
+          })}
+          ref={this.overlayRef}
+          data-element="measurementOverlay"
+        >
+          {this.shouldShowCustomOverlay(annotation) ? (
+            <CustomMeasurementOverlay
```

Comment:
I decided to move this component and `EllipseMeasurementOverlay` into the rendered component so that I don't need to import and use Draggable in three places

Reply:
I havent discussed this with Matt recently, but we may just get rid of the CustomMeasurementOverlay since we ended up implementing the EllipseMeasurement on our own.

Reply:
Makes sense to move this here, it reduces readability but I think if we remove the CustomMeasurementOverlay this will look better.

Reply:
I see. Yeah I think we can actually remove the custom overlay. I will do that in another PR.